2016-2017_Starstruck/Swerve Driver Proto/src/init.c

```diff
@@ -60,6 +60,261 @@ void initializeIO() {
  * will not start. An autonomous mode selection menu like the pre_auton() in other environments
  * can be implemented in this task if desired.
  */
+
+void pulseMotor(unsigned char x){
+   motorSet(x,127);
+   delay(250);
+   motorSet(x,-127);
+   delay(250);
+   motorSet(x,0);
+ }
+
+void manualTestSwitch(int manualCount){
```

Comment:
Maybe just call this manualTest and pull the inner switch out again, or just leave it out completely